try {
  1 div 0
} catch err:FOER0000 {
  fn:error()
}
